src/modules/timeAgo/timeAgo.ts

```diff
@@ -0,0 +1,87 @@
+import { digitsFaToEn } from "../digits";
+
+/**
+ * TimeAgo
+ *
+ * @category timeAgo
+ * @description Converting a Jalali datetime into a time ago
+ */
+class TimeAgo {
```

Comment:
Please Convert this Class to Function, We all want to migrate all Classes to Functions soon.